include/Foundation/NSURLSessionConfiguration.h

```diff
@@ -1,5 +1,7 @@
 #import <Foundation/NSObject.h>
 
 @interface NSURLSessionConfiguration : NSObject
-@end
 
+@property(class, readonly, strong) NSURLSessionConfiguration *defaultSessionConfiguration;
+
+@end
```

Comment:
So it turns out that this header file actually doesn't exist in the macOS SDK.

The interface actually lives here:
https://github.com/darlinghq/darling-foundation/blob/c952468e11a5e3c41b1213d4f64649f0b09e8dd0/include/Foundation/NSURLSession.h#L148

I think it would be a good idea to replace `+ (NSURLSessionConfiguration *)defaultSessionConfiguration;` with `@property(class, readonly, strong) NSURLSessionConfiguration *defaultSessionConfiguration;`

Reply:
Looks like this NSURLSessionConfiguration.h was not included in the build. I went ahead and removed it.